packages/rxfire/test/database.test.ts

```diff
@@ -147,7 +147,7 @@ describe('RxFire Database', () => {
           count = count + 1;
           const { event, snapshot } = change;
           expect(event).to.equal(ListenEvent.added);
-          expect(snapshot.val()).to.eql(data[snapshot.key]);
+          expect(snapshot.val()).to.eql(data[snapshot.key || '']);
```

Comment:
could it happen? Should it be `snapshot.key!` ?

Reply:
Theoretically it could be null but if it ever happens I think it's so far back in the stack that it's beyond the scope of this test to check for it. Changed to `snapshot.key!`